src/components/Player/Pages/Histograms/Histograms.jsx

```diff
@@ -19,14 +19,28 @@ const getMedian = (columns, midpoint) => {
   return medianCol && medianCol.x;
 };
 
+const getMax = (columns) => {
```

Comment:
you can do this with Math.max(...columns.map(col => col.x)).

We want the highest data point, not the bin with the most data values (that'd be the mode)